From e9a624a20ec7474614cd3a48dd98bae4f7b16a6e Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Fri, 6 Jul 2018 12:46:50 +0200
Subject: [PATCH 5/9] Edit dialog: change brush color from black to white

This looks really weird on dark color accents.

As we are supporting dynamic color accents with RRO overlays,
we might need to switch to something dynamically in future.

Change-Id: I71fccc301de43d55bf7e9a31a3166b7a59a2c236
Signed-off-by: Alexander Martinz <amartinz@shiftphones.com>
---
 res/drawable/ic_icon_change.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/drawable/ic_icon_change.xml b/res/drawable/ic_icon_change.xml
index 6495da6df..30c7227a8 100644
--- a/res/drawable/ic_icon_change.xml
+++ b/res/drawable/ic_icon_change.xml
@@ -10,7 +10,7 @@
     <item
         android:start="31dp"
         android:top="31dp">
-        <vector xmlns:android="http://schemas.android.com/apk/res/android"
+        <vector
             android:width="16dp"
             android:height="16dp"
             android:viewportHeight="24"
@@ -22,7 +22,7 @@
                 android:translateX="3"
                 android:translateY="3">
                 <path
-                    android:fillColor="#aa000000"
+                    android:fillColor="#AAFFFFFF"
                     android:pathData="M20.71,4.63L19.37,3.29C19,2.9 18.35,2.9 17.96,3.29L9,12.25L11.75,15L20.71,6.04C21.1,5.65 21.1,5 20.71,4.63M7,14A3,3 0 0,0 4,17C4,18.31 2.84,19 2,19C2.92,20.22 4.5,21 6,21A4,4 0 0,0 10,17A3,3 0 0,0 7,14Z" />
             </group>
         </vector>
-- 
2.17.1

